.github/contributors/generate.ts

```diff
@@ -1,143 +0,0 @@
-    // totally every pull or commit should account as 10
-    addContributions(5, stat.prCreators, false)
-    addContributions(5, stat.commitAuthors, false) // some commits are out pull requests
-  }
-
-  for (const stat of contributorStats) {
-    addContributions(2, stat.prCommentators, true)
-    addContributions(2, stat.issueCreators, true)
-    addContributions(2, stat.issueCommentators, true)
-    addContributions(2, stat.reviewers, true)
-    addContributions(0.3, stat.reactors, true)
-  }
-
-  return loginToTotalWeight
-}
-
-const buildContributorInfo = async (contributors: WeightedContributor[]): Promise<ContributorInfo[]> => {
-  const query = `{
-    ${contributors.map((c, i) => `user${i}: user(login: "${c.login}") {...UserFragment}`).join(`\n`)}
-  }
-    fragment UserFragment on User {
-        login
-        name
-        websiteUrl
-        avatarUrl
-    }`
-
-  try {
-    const resp = await graphql.graphql(query, {
-      headers: {
-        authorization: `token ${process.env.GITHUB_TOKEN}`,
-      },
-    })
-
-    return contributors.map((_, i) => resp[`user${i}`]).filter((v) => v)
-  } catch (err) {
-    if (err.errors && err.data) {
-      console.warn(`github errors:`, err.errors)
-      return contributors.map((_, i) => err.data[`user${i}`]).filter((v) => v)
-    }
-    throw err
-  }
-}
-
-const buildCoreTeamInfo = async (): Promise<ContributorInfo[]> => {
-  const query = `{
-        organization(login:"golangci"){
-          team(slug:"core-team"){
-            members {
-              nodes {
-                login
-                name
-                websiteUrl
-                avatarUrl
-              }
-            }
-          }
-        }
-      }`
-  const resp = await graphql.graphql(query, {
-    headers: {
-      authorization: `token ${process.env.GITHUB_TOKEN}`,
-    },
-  })
-
-  return resp.organization.team.members.nodes
-}
-
-const main = async () => {
-  try {
-    const repos = [`golangci-lint`, `golangci-lint-action`]
-    const promises = repos.map((repo) =>
-      nyc.repoContributors({
-        token: process.env.GITHUB_TOKEN,
-        user: `golangci`,
-        repo,
-        before: new Date(),
-        after: new Date(0),
-        commits: true,
-        reactions: true,
-      })
-    )
-    const contributorStats = await Promise.all(promises)
-    console.info(`Got stat for repos ${repos}`)
-    const loginToWeight = buildWeights(contributorStats)
-    const weightedContributors: WeightedContributor[] = []
-    loginToWeight.forEach((weight, login) => weightedContributors.push({ login, weight }))
-
-    weightedContributors.sort((a, b) => b.weight - a.weight)
-    const coreTeamInfo = await buildCoreTeamInfo()
-    const contributorsInfo = await buildContributorInfo(weightedContributors)
-    const exclude: any = {
-      golangcidev: true,
-      CLAassistant: true,
-      renovate: true,
-      fossabot: true,
-      golangcibot: true,
-      kortschak: true,
```

Comment:
Is this problematic? I guess the other contributors doesn't really matter since they're bots but this one specifically asked not to be mentioned. Can you exclude contributors somehow?

Reply:
I didn't find options to filter the list, but now there is no link to a profile, so it is less problematic.

Reply:
I opened https://github.com/opencollective/contributors-svg/pull/68 which works for me locally. The repo has been dead for 2,5 years so not likely they'll merge but who knows! 🤞